integration_test/gce/gce_testing.go

```diff
@@ -1106,6 +1113,30 @@ func parseImageSpec(options *VMOptions) error {
 	return nil
 }
 
+// getReleaseInfo returns the value of the requested variable in /etc/os-release.
+// For possible values, look here: https://www.freedesktop.org/software/systemd/man/latest/os-release.html
+func getReleaseInfo(ctx context.Context, logger *log.Logger, vm *VM, name string) (CommandOutput, error) {
+	cmd := fmt.Sprintf(`. /etc/os-release && echo -n $%s`, name)
+	return RunRemotely(ctx, logger, vm, cmd)
+}
+
+// getOS returns an OS struct containing information about the Operating System.
+func getOS(ctx context.Context, logger *log.Logger, vm *VM) (*OS, error) {
+	if IsWindows(vm.ImageSpec) {
+		return &OS{
+			ID: "windows",
+		}, nil
+	}
+	id_output, err := getReleaseInfo(ctx, logger, vm, "ID")
+	if err != nil {
+		return nil, err
+	}
+
+	return &OS{
+		ID: id_output.Stdout,
```

Comment:
probably want to trim any whitespace from this command output

Reply:
I ran into that earlier, and fixed it by adding `-n` to the `echo`